updater/settings.json

```diff
@@ -2,5 +2,6 @@
 "host":"github",
 "owner":"LibreHealthIO",
 "repository_name":"lh-ehr",
-"github_current":"#1113"
+"github_current":"#1113",
+"feedback_issue_number":"1182"
```

Comment:
I think that you can actually leave a trailing comma here.  Not positive, but I think arrays like that work, or are corrected or soemthing.